tools/cli/commands.go

```diff
@@ -942,7 +936,7 @@ func listOpenWorkflow(client client.Client, pageSize int, earliestTime, latestTi
 		request.TypeFilter = &s.WorkflowTypeFilter{Name: common.StringPtr(workflowType)}
 	}
 
-	ctx, cancel := newContextForLongPoll(timeout)
+	ctx, cancel := newContext(c)
```

Comment:
let's keep using longer default for list

Reply:
OK